components/basic/TabsWithTable.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from "react";
+import { Tab, Tabs, TabList, TabPanel } from "react-tabs";
+
+const TabsWithTable = ({ tabs, headers, data, filterFunction, formatDate }) => {
+  const [activeTab, setActiveTab] = useState(tabs[0]);
```

Comment:
can we name simple it as a table so we can use this type of table at every places and we can add more features here which could be anything related to table

Reply:
Sure... I think its a good idea to rename it as table.